packages/core/bootstrap/src/lib/server.ts

```diff
@@ -158,5 +158,5 @@ function initExpressMiddleware(app: express.Express) {
   })
   app.use(speedLimiter)
 
-  app.use(express.json())
+  app.use(express.json({ limit: '1mb' }))
```

Comment:
`por-indexer` kept throwing a "Payload request too large" (413) error without this. Also had to bump this limit on the actual PoR indexer service.

Reply:
Is 1MB enough (both now and in the future)? Could this impact other adapters negatively?

Reply:
I didn't actually measure how big the request payload actually was - but it may need to be increased at some point in the future if address sets get larger. I don't think this would impact other adapters? It just raises the limit for JSON payloads. If there is a way to do it just for the `por-indexer` adapter it would be ideal.

Reply:
@boxhock checked it out again, it's the WBTC endpoint sending back a ~222KB payload which goes through the json parsing middleware. So 1MB is plenty big for now - if this isn't enough in the future, we will get HTTP 413s from the `por-indexer` adapter